java/sage/FileDownloader.java

```diff
@@ -396,6 +415,27 @@ else if (remoteUIXfer)
     }
   }
 
+  private void fileTaskRun() {
+    try {
+      downloadedBytes=myDestFile.length();
```

Comment:
Clearly this is cheating since it's not reporting real-time progress...but I'm fine with it. :)